ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -180,7 +123,6 @@ nnfw_open (const GstTensorFilterProperties * prop, void **private_data)
   int err = 0;
   nnfw_pdata *pdata;
   char *model_path = NULL;
-  char *meta_file = NULL;
```

Comment:
One question. After this PR, how can we get a meta-file name for the "nnfw" tensor filter?

Reply:
nnstreamer doesn't need to verify the meta. This is up to nnfw library.
If metadata is missing, nnfw will return error now.

Now to run nnfw, metadata is necessary. However, nnfw team considers to remove meta file or other option.
So he requested to remove this validation, and to handle directory path as model property.